packages/ui/hooks/useUSDPrices.ts

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import { useQuery } from 'react-query';
+
+async function getUSDPriceOf(cgIds: string[]): Promise<number[]> {
```

Comment:
Shouldn't we fetch it from supabase?

Reply:
Yea maybe, not sure yet. It's a WIP i wanted to include. 
It depends on the data we get from the call. Trying to split these up into reasonable chunks.